src/app/components/select-field/index.html

```diff
@@ -40,63 +40,70 @@
+    >
+      Indicates whether to use the default search function. To provide search results from a remote source, consumers can set this property to <sky-code>false</sky-code> and specify the source with the <sky-code>data</sky-code> property. This property accepts <sky-code>boolean</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="multipleSelectOpenButtonText"
       isOptional="true"
       defaultValue="Select values"
     >
-    Specifies a label for the button when <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>. This property accepts <sky-code>string</sky-code> values.
+      Specifies a label for the button when <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="pickerHeading"
       isOptional="true"
     >
-    Specifies a header for the select field component's modal. When <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, the default header is "Select a value." When <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>, the default header is "Select values." This property accepts <sky-code>string</sky-code> values.
+      Specifies a header for the picker. When <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, the default header is "Select a value." When <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>, the default header is "Select values." This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="selectMode"
       isOptional="true"
       defaultValue="multiple"
     >
-    Specifies the selection mode that determines whether users can select one item or multiple items in the select field component's modal. The valid options are <sky-code>single</sky-code>, which displays a text input and allows users to select one item, and <sky-code>multiple</sky-code>, which displays a button and allows users to select multiple items.
+      Specifies the selection mode that determines whether users can select one item or multiple items. The valid options are <sky-code>single</sky-code>, which displays a text input, and <sky-code>multiple</sky-code>, which displays a button. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="showAddNewRecordButton"
       isOptional="true"
       defaultValue="false"
     >
-    Indicates whether to display a button for users to add new items directly from the select field modal. You must provide the logic for adding items.
+      Indicates whether to display a button in the picker for users to add items. Consumers must provide the logic to add items. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
In the interest of further improvement, should we also mention that consumers would need to tie into the `addNewRecordButtonClick` event to run their custom logic?

Reply:
Good call. Making an update now and will have it in place shortly...